pkg/i18n/english.go

```diff
@@ -1894,6 +1895,7 @@ func EnglishTranslationSet() *TranslationSet {
 		CommandDoesNotSupportOpeningInEditor:     "This command doesn't support switching to the editor",
 		CustomCommands:                           "Custom commands",
 		NoApplicableCommandsInThisContext:        "(No applicable commands in this context)",
+		SelectCommitsOfCurrentBranch:             "Select commits of current branch",
```

Comment:
Wonder if it would be clearer to say 'Select commits from the head of the current branch to the last merge commit' because technically all commits to the root are commits of the current branch. To be honest though I'd rather go with the simpler approach and see if people get confused about it than the other way around.